go/app/main.go

```diff
@@ -33,24 +55,143 @@ func addItem(c echo.Context) error {
 	message := fmt.Sprintf("item received: %s", name)
 	res := Response{Message: message}
 
+	//ここからStep2
+	//jsonファイルの読み込み
+	raw, err := os.ReadFile("./items.json")
+	//エラーハンドリング
+	if err != nil {
+        fmt.Println("Read error")
```

Comment:
ここでerrorが出た場合に、続きの処理をするとずっとエラーが出るのでエラーが出たらreturnするようにしましょう。他の部分も同様です。
```suggestion
        return err
```

または

```suggestion
        return fmt.Errorf("Read error: %v", err) // 勘で書いてるので構文間違ってたら直してください🙇‍♀️
```

ステータスコードを設定したいなら

```suggestion
        return c.JSON(http.StatusInternalServerError, Response{Message: err.Error()})
```

これでもいいです。errorの型を返すようにしてDebugできるようにしたいです。